dags/sentinel-2_data_transfer.py

```diff
@@ -245,7 +238,21 @@ def start_transfer(stac_item):
             else:
                 copied_files.append(result)
 
-    if len(copied_files) == 17:
+    # write the STAC file to s3
+    s3_hook = S3Hook(aws_conn_id=AFRICA_CONN_ID)
+    if s3_hook.list_keys(DEST_BUCKET_NAME, prefix=scene_path) == 17:
```

Comment:
This will be slow...

Do you think it's necessary? Or should we just count here?

Reply:
Does it need a `len()` too?

Reply:
It's not really necessary, I'm tightening the net.   We can check the count of copied_files before copying the STAC and that should be enough.